UPGRADING.md

```diff
@@ -14,7 +16,6 @@ upgrading IRIDA that cannot be automated.
 
 0.22.0 to 19.01
 ----------------
-* The following new Tomcat variable should be set for deployment `irida.db.profile=prod` for production deployments. See https://irida.corefacility.ca/documentation/administrator/web/#servlet-container-configuration for more details.
```

Comment:
I think this is being mistakenly deleted?

Reply:
Merge artifact.  Fixed in d4868f7